include/boost/math/special_functions/prime_functions.hpp

```diff
@@ -0,0 +1,106 @@
+// Copyright 2020 Matt Borland
+//
+// Use, modification and distribution are subject to the
+// Boost Software License, Version 1.0.
+// (See accompanying file LICENSE_1_0.txt
+// or copy at http://www.boost.org/LICENSE_1_0.txt)
+
+#ifndef BOOST_MATH_SPECIAL_FUNCTIONS_PRIME_FUNCTIONS_HPP
+#define BOOST_MATH_SPECIAL_FUNCTIONS_PRIME_FUNCTIONS_HPP
+
+#include <boost/math/special_functions/prime.hpp>
```

Comment:
Why is this header needed?

Reply:
@NAThompson For small values (<= 104729) `prime_range` does not call the sieve, but uses the lookup table found in that header file.

Reply:
@mborland : In the _Joy of Factoring_, the author states that sieves are computed faster than the can be read out of program memory.

In addition, you deprive yourself of a great unit test, namely checking that table.

Reply:
Here's a link: https://bookstore.ams.org/stml-68

Reply:
@NAThompson That is interesting. I'll have to give it a read.